_database-integrations/amazon-s3/amazon-s3-csv-latest.md

```diff
@@ -0,0 +1,289 @@
+
+          To ensure data is correctly captured, you'd want to enter a search pattern that would match all files beginning with `customer`, regardless of the date in the file name. This would map all files in the `analytics` folder that begin with `customers` to a single table:
+
+          ```
+          /analytics/customers.*\csv
+          ```
+
+          This search pattern would match `customers-2018-07-01.csv`, `customers-2018-07-02.csv`, `customers-2018-07-03.csv`, etc., and ensure files are replicated as they're created or updated.
+
+      - title: "Define the table's Primary Key"
+        anchor: "define-table-primary-key"
+        content: |
+          {% include note.html type="single-line" content="This step is optional." %}
+
+          In the **Primary Key** field, enter one or more header fields (separated by commas) Stitch can use to identify unique rows. For example:
+
+          ```
+          account_id,date
+          ```
+
+          If undefined, Stitch will load data into the table in an append-only fashion. This means that existing rows in the destination won't be updated, but will be appended to the end of the table. Refer to the [Primary Keys and Append-Only Replication](#primary-keys-append-only) section below for more info and examples.
+
+      - title: "Specify datetime fields"
+        anchor: "specify-datetime-fields"
+        content: |
+          {% include note.html type="single-line" content="This step is optional." %}
+
+          In the **Specify datetime fields** field, enter one or more header fields (separated by commas) that should appear in the destination table as `datetime` fields instead of strings. For example:
+
+          ```
+          created_at,updated_at
+          ```
+
+          If undefined, Stitch will load all columns into the destination as strings.
+
+      - title: "Configure additional tables"
+        anchor: "configure-additional-tables"
+        content: |
+          If you want to add another table, click the **Configure another table?** link below the **Specify datetime fields** field. Otherwise, move onto the [Sync historical data](#define-historical-sync) section.
+
+           Stitch doesn't enforce a limit on the number of tables that you can configure for a single integration.
+
+  - title: "historical sync"
+    ## For this, we should note that setting this date will replicate all files in full that have been modified since the date set here
+
+    content: |
+      For example: Let's say we've added a `customers.*\csv` search pattern and set the integration's historical **Start Date** to 1 year. During the initial replication job, Stitch will replicate the contents of all files that match the search pattern that have been modified in the past year.
```

Comment:
I don't think you want the `\` here.

Reply:
Yeah - this was from the examples you gave me yesterday. Shouldn't it be `analytics/customers\.*csv` so the `.` is escaped?